tests/test_end_to_end.py

```diff
@@ -0,0 +1,95 @@
+"""
+End-to-end tests for each backend configuration. Sets up a server with
+the backend, sends some basic queries to that server and verifies results
+are as expected.
+"""
+import unittest
+
+from ga4gh.backends import WormtableBackend
+from tests.test_backends import WormtableTestFixture
+import ga4gh.protocol as protocol
+import ga4gh.server as server
+
+
+class EndToEndTest(unittest.TestCase):
+    def setUpServer(self, backend):
+        server.app.config['TESTING'] = True
+        server.app.config["VariantBackend"] = backend
+        self.app = server.app.test_client()
+
+    def sendJSONPostRequest(self, path, data):
+        return self.app.post(path,
+                             headers={'Content-type': 'application/json'},
+                             data=data)
+
+    def runVariantSetsSearch(self):
+        expectedIds = ['example_1', 'example_2', 'example_3', 'example_4']
```

Comment:
This is fine for now, but we should modify our test backend API to return the variant sets we want so we don't need to hard code these values. Could you insert a TODO comment to this effect here please?